tests/test_minimal/test_utils/test_json_schema_utils.py

```diff
@@ -39,35 +35,6 @@ def sort_item(item):
         return item
 
 
-def test_get_schema_from_method_signature():
-    class A:
-        def __init__(self, a: int, b: float, c: Union[Path, str], d: bool, e: str = "hi", f: Dict[str, str] = None):
-            pass
-
-    schema = get_schema_from_method_signature(A.__init__)
-
-    correct_schema = dict(
-        additionalProperties=False,
-        properties=dict(
-            a=dict(type="number"),
-            b=dict(type="number"),
-            c=dict(type="string"),
-            d=dict(type="boolean"),
-            e=dict(default="hi", type="string"),
-            f=dict(type="object", additionalProperties={"^.*$": dict(type="string")}),
```

Comment:
I was looking for a reference to the regex-like expression here in f in the new tests. Is this something we are changing?

Reply:
If you read the regex carefully it's not saying anything different from the current form (both say 'a string, including the empty string'). According to tests does not alter any validation conditions either

Though if anything the regex ought to be `*.+$` to indicate that each key in the dictionary must have at least one character

Reply:
Good. To be honest, I don't know regex enougn and I always have to read it. I was pointing it out in case it was a change by omission instead of by intention. If the code does the same and we don't need to put regex on it I think that's a win.